### ISSUE TITLE
Can't submit a job from the Editor

### ISSUE THREAD

**Issue by skurnevich** [zowe/zlux#951]
Job submitting feature seems to be broken in the Editor app.
On submit click nothing happens and error pops-up in console output.

The issue seems to be related to [this line](https://github.com/zowe/zlux-editor/blob/v2.x/staging/webClient/src/app/core/menu-bar/menu-bar.config.ts#L59) particularly.
`let content = context.editor.model.getValue();`
`context.editor.model` is undefined, but `context.editor.getValue()` returns editor content, probably editor interface was updated some time ago but hasn't been reflected in the job submitting function. 

**Screenshots (if needed)**
![Screenshot 2022-11-28 at 18 00 36](https://user-images.githubusercontent.com/61471197/204359996-61dfec59-34c7-45dc-b868-f9d2cc7864c3.png)

**Comment by 1000TurquoisePogs** [comment]
@DivergentEuropeans was working on this at one point.
It requires the jes explorer to exist and you need to access from the gateway.
Other than that, not sure if this is recently fixed or recently broke.

**Comment by skurnevich** [comment]
It looks like it's not related to the jes explorer or accessing from the gateway, as both conditions are met for me, but you're right, now i see that Leanid has fixed a [similar issue](https://github.com/zowe/zlux-editor/pull/293) for v1. 

Job submission started working in 1.28.1 again, however, there is still a minor issue, it can't find the Job ID afterward 
![Screenshot 2022-11-29 at 11 37 36](https://user-images.githubusercontent.com/61471197/204508799-f66f770f-7bf9-4de2-8c31-4eb5583f629c.png)

So basically same change need to be done for v2, @DivergentEuropeans do you have it in your plans? I can also take this if you are busy with something else.

**Comment by DivergentEuropeans** [comment]
@skurnevich Ugh! I didn't see the notification for this apologies. It is not in my plans atm, I'm busy with a few things - fixed in V1 as noted, not in V2 (API is different)

**Comment by DivergentEuropeans** [comment]
I tried to do it in V2, some months ago when I had a little extra time, but the API was different

**Comment by cfl0000** [comment]
Good day:
  I have the same issue on version 2.15, when it will be fixed please?

**Comment by DivergentEuropeans** [comment]
@cfl0000 Hey, this isn't on the map atm as the squad continues working hard to wrap the Techical Preview for the Zowe Server Install Wizard + doing intense Angular research for V3

What's your priority?

**Comment by cfl0000** [comment]
It should be a medium, thanks

在 2024年6月23日週日 14:12，Leanid Astrakou ***@***.***> 寫道：
